lib/kv_storage_redis.ex

```diff
@@ -1,18 +1,116 @@
 defmodule KvStorageRedis do
   @moduledoc """
-  Documentation for `KvStorageRedis`.
+  Модуль для работы с БД Redis
   """
 
+  use GenServer
+
+  defstruct host: nil, port: nil, db: nil, conn: nil
```

Comment:
Тип не помешает для тех, кто будет читать код.
```suggestion
  @typedoc "…"
  @type t :: %{
    __struct__: __MODULE__,
    host: binary(),
    port: pos_integer(),
    db: non_neg_integer(),
    conn: any()
  }
  
  defstruct host: nil, port: nil, db: nil, conn: nil
```